draft-trammell-taps-post-sockets.md

```diff
@@ -435,14 +436,24 @@ reuse an Association if they can be carried from the same Local to the same
 Remote over the same Paths; this re-use of an Association may implies the
 creation of a new Transient.
 
+Associations may exist without a Message Carrier if required. This may be done if
+peer cryptographic state, e.g., a pre-shared key, is established out-of-band.
+Thus, Associations may be created without the need to send application data
+to a peer, i.e., without a Remote. Associations may also be created from
+existing Associations. In doing so, some state related with an Association may mutate.
+For example, a pre-shared key may be extended with new Association-specific information for
+diversification purposes. This allows new Associations and their Transients to
+be quickly created without performing in-band cryptographic handshakes.
+See {{I-D.ietf-taps-crypto-sep}} for more details about this separation.
+
 ## Remote
 
 A Remote represents information required to establish and maintain a
 connection with the far end of an Association: name(s), address(es), and
 transport protocol parameters that can be used to establish a Transient;
-transport protocols to use; information about public keys or certificate
+transport protocols to use; trust model information about public keys or certificate
 authorities used to identify the remote on connection establishment; and so
-on. Each Association is associated with a single Remote, either explicitly by
+on. Each Remote is bound to an Association, either explicitly by
```

Comment:
Any particular reason to reverse this? I think the previous language was more specific?